README.md

```diff
@@ -1,27 +1,133 @@
-This repository contains:
+**Table of Contents**
 
-+ Latest Windows agent source code and binaries -- For handling user account and address management.
-+ Latest version of the metadata scripts source code and binary -- For handling startup and shutdown scripts.
-+ Latest sysprep scripts -- For running sysprep on new Windows virtual machines.
-+ Latest VSS agent -- For handling VSS on snapshots.
+* [Background](#background)
+* [Agent](#Agent)
+    * [Account Setup](#account-setup)
+    * [IP Forwarding](#ip-forwarding)
+* [Instance Setup](#instance-setup)
+* [Metadata Scripts](#metadata-scripts)
+* [Network Setup](#network-setup)
+* [Packaging and Package Distribution](#packaging-andpackage-distribution)
+* [Contributing](#contributing)
+* [License](#license)
 
-## Contribute
+## Background
 
-Have a patch that will benefit this project? Awesome! Follow these steps to have it accepted.
+The Windows guest environment is the Google provided configuration and
+tooling inside of a [Google Compute Engine](https://cloud.google.com/compute/)
+(GCE) virtual machine. The
+[metadata server](https://cloud.google.com/compute/docs/metadata) is a
+communication channel for transferring information from a client into the guest.
+The Windows guest environment includes a set of scripts and binaries that read 
+the content of the metadata server to make a virtual machine run properly on 
+Google Compute Engine.
 
-1. Please sign our [Contributor License Agreement](CONTRIB.md).
-1. Fork this Git repository and make your changes.
-1. Create a Pull Request
-1. Incorporate review feedback to your changes.
-1. Accepted!
+## Agent
 
-## License
+#### Account Setup
+
+The agent handles [creating user accounts and setting/resetting passwords](https://cloud.google.com/compute/docs/instances/windows/creating-passwords-for-windows-instances).
+
+#### IP Forwarding
+
+The agent uses IP forwarding metadata to setup or remove IP routes.
+
+*   Only IPv4 IP addresses are currently supported.
+
+## Instance Setup
+
+`instance_setup.ps1` is configured by GCE sysprep to run on VM first boot. 
+The script performs the following tasks:
+
+*   Set the hostname to the the instance name.
+*   Runs user provided 'specialize' startup script.
+*   Activates Windows using a KMS server.
+*   Sets up RDP and WinRM to allow remote login.
+
+## Metadata Scripts
+
+Metadata scripts implement support for running user provided
+[startup scripts](https://cloud.google.com/compute/docs/startupscript) and
+[shutdown scripts](https://cloud.google.com/compute/docs/shutdownscript).
+
+## Packaging and Package Distribution
+
+The guest code is packaged in [GooGet](https://github.com/google/googet)
+packages and published to Google Cloud repositories.
 
-All files in this repository are under the [Apache License, Version 2.0](LICENSE) unless noted otherwise.
+We build and install the following packages for the Windows guest environment:
+  
+*   `google-compute-engine-windows` 
+    *   Windows agent exe
```

Comment:
Can you drop the internal bullets and make them sentences?

*   `google-compute-engine-windows` - Windows agent executable.